spec/features/bulkrax_import_spec.rb

```diff
@@ -274,7 +270,7 @@
           let(:collection_2) { create :collection, id: "bedd7330-5040-4687-8226-0851f7256dff", title: ["This Title Should Change"] }
 
           it "updates the collections" do
-            perform_enqueued_jobs(only: [Bulkrax::ImporterJob, HykuAddons::ImportWorkCollectionJob]) do
+            perform_enqueued_jobs(only: [Bulkrax::ImporterJob, Bulkrax::ImportWorkCollectionJob]) do
```

Comment:
`Bulkrax::ImportWorkCollectionJob` shouldn't be called here, the HykuAddons one should.  Our one sets collection titles, and the Bulkrax ones doesn't.  If the HykuAddons one isn't being called any more we have a problem.